docs/collator/SetupAndRun/sync.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 5
+sidebar_position: 4
```

Comment:
I attempted to switch the sync and keys steps in the sidebar.
Lmk if you see sync before keys, i might have some caching problem, it's still in the old order when I open the vercel

Reply:
using autogenerated sidebar now. fixed